internal/provider/diagnostics.go

```diff
@@ -20,3 +20,16 @@ func randomReadError(errMsg string) diag.Diagnostics {
 
 	return diags
 }
+
+func hashGenerationError(errMsg string) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	diags.AddError(
+		"Hash Generation Error",
+		"While attempting to generate a hash from of the password an error occurred.\n\n"+
+			"Verify that the state contains a populated 'result' field and retry the operation\n\n"+
```

Comment:
Might be worth mentioning the `terraform state show` command. 👍

Reply:
Added comment to use `terraform state show`.